.jshintrc

```diff
@@ -9,7 +9,7 @@
 	"curly": false, // Require {} for every new block or scope.
 	"eqeqeq": true, // Require triple equals i.e. `===`.
 	"immed": true, // Require immediate invocations to be wrapped in parens e.g. `( function(){}() );`
-	"latedef": true, // Prohibit variable use before definition.
+	"latedef": "nofunc", // Prohibit variable use before definition.
```

Comment:
This change is to support angular best practices (as implemented in templates.js).  Taken from [angular-styleguide](https://github.com/johnpapa/angular-styleguide#style-y230)